dashboard/app/app.yaml

```diff
@@ -15,15 +15,15 @@ inbound_services:
 
 handlers:
 - url: /favicon.ico
-  static_files: static/favicon.ico
-  upload: static/favicon.ico
+  static_files: dashboard/app/static/favicon.ico
```

Comment:
Will these be served fine in the GOPATH mode?

Reply:
No. Do you think we need GOPATH support?

Reply:
In GOPATH AppEngine considers dashboard/app as a project root.

Reply:
Can we run dashboard tests so that CWD is `syzkaller/`?
If yes, we could probably simplify `Glob*` functions.

Reply:
It is the least invasive approach I can think about.
dashboard/app is a package and it looks correct to consider package dir a CWD in its own tests. Additional magic will make it confusing.
I'm adding changes to the part with the smallest interest (I think we are the only who use AppEngine part).

PS. The things to remember here:
The problem is the way we use AppEngine.
app.yaml and go.mod are expected to be in the same folder. Also, the App can't be be in GOPATH, but it is in out case.
The correct way is to move the App out of GOPATH. In our case it is a quite sensitive project structure change.

Reply:
That's interesting, thanks for the clarification.

I wonder if we could move `app.yml` to the root of our package without breaking anything. From the docs it looks like it's exactly the purpose of the `main` field:

https://cloud.google.com/appengine/docs/standard/reference/app-yaml?tab=go

Reply:
Adding "main:" + moving app.yml to syzkaller/ got the same problem.
File operations believe syzkaller/ is a root.
The option "main:" adds the next line to log:
"runtimeMainExecutablePath": "github.com/google/syzkaller/dashboard/app"
but it doesn't change the CWD.